### PR TITLE
Restart WATT Task after image push

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] N/A
[Problem] User has to manually restart WATT Task using AWS Console
[Solution] Modify the script to do this automatically.

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

File: watt-aws-update.sh [hunk 1]
```diff
@@ -61,3 +61,33 @@ if [ $? -ne 0 ]; then
 fi
 
 echo -e "\033[0;32mSuccessfully pushed watt image to \033[0m"$watt_docker_aws_repository_uri
+
+# Install jq package.
+if ! dpkg -s jq; then
+    if ! sudo apt-get install -y jq; then
```
Comment (gwolny) [inline watt-aws-update.sh]: What about just checking if package jq is installed.
If not just print command to install and exit. And such check should be made at the script beginning.

Comment (grzegorz-czajkowski) [reply watt-aws-update.sh]: Done.

**Comment by grzegorz-czajkowski** [comment]
Patch works and can be reviewed.

**Comment by grzegorz-czajkowski** [comment]
The build has been terminated while downloading emscripten. Please ignore continuous-integration verification.

**Pull request merged**